realm/realm-library/src/androidTest/java/io/realm/internal/TableIndexAndDistinctTest.java

```diff
@@ -85,26 +132,37 @@ public void shouldTestIndexOnWrongColumnType(Long index) {
         t.addSearchIndex(index);
     }*/
 
-    public void testShouldCheckIndexIsOkOnColumn() {
+    @Test
+    public void shouldCheckIndexIsOkOnColumn() {
         init();
+        sharedRealm.beginTransaction();
```

Comment:
can we call `beginTransaction()` in the `setUp()`  and `cancelTransaction()` in the `tearDown()`?